xain/CONFIG.md

```diff
@@ -0,0 +1,18 @@
+# Configuration
+Currently configuration happens two fold.
+
+## `config.py`
+On the initial run of the library the `xain.config.init_config()` function
+will be executed and initialize a `config.cfg` file in the projects root directory. 
```

Comment:
should this be `config.py`?

Reply:
Nope. the `config.py` resides in `xain/config.py`.  In the top level `init.py` we call xain.config.init_config() if no `config.cfg` is yet present in the top level directory and create it in that case.